package.json

```diff
@@ -66,7 +66,7 @@
     "chai-fs": "^1.0.0",
     "chai-subset": "^1.6.0",
     "core-js": "^3.0.0",
-    "eslint": "^5.15.2",
+    "eslint": ">=5.15.2",
```

Comment:
Should we do a “5 || 6” type of logic here? We aren’t necessarily compatible with some future version 7.

Reply:
That's true, let's go for it.

Reply:
I think it would be better to keep `>=5.15.2`:

This is a dev dependency and we don't enforce `eslint` version so what we put here doesn't really matter except for our tests. It would help spot an incompatibility once v7 is released.